src/server.cr

```diff
@@ -85,10 +84,6 @@ module Sparoid
       end
     end
 
-    private def verify_ip(ip_str, client_addr)
-      ip_str == client_addr.address || raise "source ip doesn't match"
```

Comment:
Should we still have some logging? When the source is different? So we have some sort of audit trail